caesar_cipher/lib/cipher.rb

```diff
@@ -0,0 +1,35 @@
+class Cipher
+  LOW_ALPHABET = ("a".ord.."z".ord)
+  HIGH_ALPHABET = ("A".ord.."Z".ord)
+  ALPHABET_LENGHT = 26
+
+  def initialize(rot_key)
+    if rot_key > ALPHABET_LENGHT || rot_key < 0
+      raise ArgumentError, "Number must be between 0 and 26"
+    else
```

Comment:
@zzokki81 no need to have an `else` clause. Once `raise` is called the method stops executing.